tmail-backend/combined-identity/src/main/java/com/linagora/tmail/combined/identity/CombinedUsersRepositoryModule.java

```diff
@@ -34,6 +38,8 @@ public void configure() {
 
         bind(UsersDAO.class).to(CombinedUserDAO.class);
         bind(UsersRepository.class).to(CombinedUsersRepository.class);
+        bind(DelegationStore.class).to(CassandraDelegationStore.class);
+        bind(Authorizator.class).to(DelegationStoreAuthorizator.class);
```

Comment:
How about LDAP?

Reply:
I thought that LDAP would use LdapUsersRepositoryModule (NaiveDelegationStore and UserRepositoryAuthorizator) same as James side?